### ISSUE TITLE
All Good, But Errors on !fix Command

### ISSUE THREAD

**Issue by Dinli800** [715209/nginx-obs-automatic-low-bitrate-switching#54]
Hello, I installed the new version 1.9.2. All commands run, stream and switch work. Only that! Fix doesn't work. I upload error picture. And I also had a picture from earlier version 1.6. and 1.7. and no problems with it. Does anyone have any idea where to look for errors?

![1](https://user-images.githubusercontent.com/78984511/107823809-b384d300-6d80-11eb-946d-5c8566873257.JPG)
![2](https://user-images.githubusercontent.com/78984511/107823817-b5e72d00-6d80-11eb-8617-7da0b26826a0.JPG)

**Comment by 715209** [comment]
﻿Hello Dinli800,

Thank you for reporting this bug.
I am still working on this version, you can download the latest release here: https://github.com/715209/nginx-obs-automatic-low-bitrate-switching/releases .

**Comment by Dinli800** [comment]
Hi,
Thanks for the quick answer, I have now loaded Release 1.9.1, after "npm install --production" this version works again with all commands. Excellent! Only I noticed it is 1.9.0 not 1.9.1, should this download be like that? 
![566456456456456](https://user-images.githubusercontent.com/78984511/107851189-487edf00-6e08-11eb-9bc7-de71a6f97683.JPG)
![546654465465446545](https://user-images.githubusercontent.com/78984511/107851190-4a48a280-6e08-11eb-9e46-071aadbac11e.JPG)
